web/src/api/APIClient.ts

```diff
@@ -163,51 +274,61 @@ export const APIClient = {
+      `api/notification/${notification.id}`,
+      { body: notification }
+    ),
     delete: (id: number) => appClient.Delete(`api/notification/${id}`),
-    test: (n: Notification) => appClient.Post("api/notification/test", n)
+    test: (notification: ServiceNotification) => appClient.Post("api/notification/test", {
+      body: notification
+    })
   },
   release: {
     find: (query?: string) => appClient.Get<ReleaseFindResponse>(`api/release${query}`),
     findRecent: () => appClient.Get<ReleaseFindResponse>("api/release/recent"),
-    findQuery: (offset?: number, limit?: number, filters?: Array<ReleaseFilter>) => {
-      const params = new URLSearchParams();
-      if (offset !== undefined && offset > 0)
-        params.append("offset", offset.toString());
-
-      if (limit !== undefined)
-        params.append("limit", limit.toString());
+    findQuery: (offset?: number, limit?: number, filters?: ReleaseFilter[]) => {
+      const params: Record<string, string[]> = {
+        indexer: [],
+        push_status: [],
+        q: []
+      };
 
       filters?.forEach((filter) => {
         if (!filter.value)
           return;
 
-        if (filter.id == "indexer")
-          params.append("indexer", filter.value);
-        else if (filter.id === "action_status")
-          params.append("push_status", filter.value);
-        else if (filter.id == "torrent_name")
-          params.append("q", filter.value);
+        if (filter.id == "indexer") {
+          params["indexer"].push(filter.value);
+        } else if (filter.id === "action_status") {
+          params["push_status"].push(filter.value);
+        } else if (filter.id == "torrent_name") {
+          params["q"].push(filter.value);
+        }
       });
 
-      return appClient.Get<ReleaseFindResponse>(`api/release?${params.toString()}`);
+      return appClient.Get<ReleaseFindResponse>("api/release", {
+        queryString: {
+          offset,
+          limit,
+          ...params
+        }
+      });
     },
     indexerOptions: () => appClient.Get<string[]>("api/release/indexers"),
     stats: () => appClient.Get<ReleaseStats>("api/release/stats"),
-    delete: (olderThan: number) => {
-      const params = new URLSearchParams();
-      if (olderThan !== undefined && olderThan > 0) {
-        params.append("olderThan", olderThan.toString());
-      }
-
-      return appClient.Delete(`api/release?${params.toString()}`)
-    },
-    replayAction: (releaseId: number, actionId: number) => appClient.Post(`api/release/${releaseId}/actions/${actionId}/retry`)
+    delete: (olderThan: number) => appClient.Delete("api/release", {
+      queryString: { olderThan }
+    }),
+    replayAction: (releaseId: number, actionId: number) => appClient.Post(
+      `api/release/${releaseId}/actions/${actionId}/retry`
+    )
   },
   updates: {
     check: () => appClient.Get("api/updates/check"),
-    getLatestRelease: () => appClient.Get<GithubRelease | undefined>("api/updates/latest")
```

Comment:
Someone should confirm here whether this `| undefined` part really was unintended or whether it was just legacy junk.

Reply:
Probably unintended or needed during dev, but the endpoint should always get the latest version.

Reply:
The endpoint will return nothing (no-content) if running the latest. I belive it is considered undefined